packages/arb-token-bridge-ui/src/hooks/useETHPrice.ts

```diff
@@ -12,10 +12,10 @@ export type UseETHPriceResult = {
 
 export function useETHPrice(): UseETHPriceResult {
   const { data, error, isValidating, mutate } = useSWR<number, Error>(
-    'https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=usd',
-    url => axios.get(url).then(res => res.data.ethereum.usd),
+    'https://api.coinbase.com/v2/prices/ETH-USD/spot',
+    url => axios.get(url).then(res => res.data.data.amount),
     {
-      refreshInterval: 30_000,
+      refreshInterval: 300_000,
```

Comment:
As per description - `This PR changes the refresh interval form every 30s to every 1 min`
This is 5 minutes. Are we sure about it?

Reply:
Ah got it, it was a separate thread.